pkg/dbutil/types.go

```diff
@@ -23,3 +23,11 @@ func IsFloatType(tp byte) bool {
 
 	return false
 }
+
+// IsTimeType returns true if tp is time type
+func IsTimeType(tp byte) bool {
+	if tp == mysql.TypeDatetime || tp == mysql.TypeTimestamp || tp == mysql.TypeDate {
```

Comment:
what about `TypeNewDate ` and `TypeDuration`?

Reply:
only these three type encoded in tidb bucket. This function's name is not suitable, I rename it.